client/jetpack-onboarding/summary-completed-steps.jsx

```diff
@@ -5,8 +5,9 @@
  */
 import React from 'react';
 import classNames from 'classnames';
+import PropTypes from 'prop-types';
```

Comment:
Nit: Some folks prefer `PropTypes` is to be another exception for the 🔤  rules, keeping it right below the `React` one. I don't really mind keeping it as-is though.